### PR TITLE
Switch CAPA CI jobs to use pod utils as bootstrap.py is deprecated

### PR DESCRIPTION
**Pull request by dims**
This went easier since we made similar changes in CAPG ( 🤞 )

### CODE REVIEW

File: config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml [hunk 1]
```diff
@@ -148,29 +153,34 @@ periodics:
     preset-aws-credential: "true"
   branches:
     - release-0.4
+  decorate: true
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api
+    base_ref: master
```
Comment (detiber) [inline config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml]: Should this be `release-0.2` here instead?

Comment (detiber) [reply config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml]: Or is capi even needed here? it wasn't listed in the repos removed below

File: config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml [hunk 2]
```diff
@@ -97,29 +97,34 @@ periodics:
     preset-aws-credential: "true"
   branches:
     - master
+  decorate: true
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api
+    base_ref: master
+    path_alias: "sigs.k8s.io/cluster-api"
```
Comment (detiber) [inline config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml]: Is CAPI needed here? It wasn't in the repos removed below.

File: config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml [hunk 3]
```diff
@@ -185,6 +195,63 @@ periodics:
...
+    preset-bazel-remote-cache-enabled: "true"
+    preset-service-account: "true"
+    preset-aws-ssh: "true"
+    preset-aws-credential: "true"
+  branches:
+    - release-0.4
+  decorate: true
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api
```
Comment (detiber) [inline config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml]: Is the CAPI repo needed here?

File: config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml [hunk 4]
```diff
@@ -85,33 +85,35 @@ presubmits:
       preset-aws-credential: "true"
       preset-bazel-scratch-dir: "true"
       preset-bazel-remote-cache-enabled: "true"
+    path_alias: "sigs.k8s.io/cluster-api-provider-aws"
     always_run: true
     optional: true
+    decorate: true
+    extra_refs:
+    - org: kubernetes-sigs
+      repo: cluster-api
```
Comment (detiber) [inline config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml]: Is the CAPI repo needed? Looks like the previous ci jobs weren't using it.

File: config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml (hunk 1 above)
Comment (dims) [reply config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml]: sorry, deleted

File: config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml (hunk 3 above)
Comment (dims) [reply config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml]: sorry, deleted

File: config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml (hunk 4 above)
Comment (dims) [reply config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml]: sorry, deleted

File: config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml (hunk 2 above)
Comment (dims) [reply config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml]: sorry, deleted

**Pull request merged**